app/models/bill.rb

```diff
@@ -0,0 +1,2 @@
+class Bill < ApplicationRecord
```

Comment:
In the migration, you have foreign keys in all of the tables. Those relationships need to be included in all of the models as well.

Reply:
I did not do anything with the models just the migration files because the team was going to take care of that I had to leave for my other job at 2:00PM

Reply:
It's fine that you weren't able to get to it today, but pull requests in general should be as complete of a feature as possible so that all of the code related to a specific issue is bundled nicely together.

Reply:
We are using the pull request as a way to share workload, the sooner they get the migration they can do what they would like with the other files.

Reply:
Okay. Did you see my main comment above? 

For all of the changes that you want to make to the data structure, you need to actually create new migrations instead of editing or deleting the migrations that had already been incorporated into the master branch. Please make these changes by creating new migration files.

See https://guides.rubyonrails.org/active_record_migrations.html#changing-existing-migrations.